### DISCUSSION TITLE
How to schedule time for a query

### DISCUSSION THREAD

**Discussion by karishma-ag** [getredash/redash#6058] [Q&A]
With redash v10.1.0 there isn't an option to choose time to run at a certain time of the day as it's mentioned on the doc [here](https://redash.io/help/user-guide/querying/scheduling-a-query)
Since the option isn't there currently what time does it choose as the starting point ? 

Here is the screenshot of how does the refresh schedule settings look for us
<img width="596" alt="Screenshot 2023-05-23 at 12 39 44 PM" src="https://github.com/getredash/redash/assets/119491183/499e6c44-6f68-46b1-b8e1-cb909f39eef8">

**Comment by gaecoli** [comment]
> With redash v10.1.0 there isn't an option to choose time to run at a certain time of the day as it's mentioned on the doc [here](https://redash.io/help/user-guide/querying/scheduling-a-query) Since the option isn't there currently what time does it choose as the starting point ?
> 
> Here is the screenshot of how does the refresh schedule settings look for us <img alt="Screenshot 2023-05-23 at 12 39 44 PM" width="596" src="https://user-images.githubusercontent.com/119491183/240166026-499e6c44-6f68-46b1-b8e1-cb909f39eef8.png">

You should choose 1days and set day time.

**Comment by gaecoli** [comment]
<img width="458" alt="image" src="https://github.com/getredash/redash/assets/46274612/bf878282-706b-40b0-948c-f94c7601cf1a">
<img width="550" alt="image" src="https://github.com/getredash/redash/assets/46274612/05ac6a91-6401-4dac-b7de-7a20a1b92fb8">
